index.html

```diff
@@ -39,21 +39,113 @@
       rel="stylesheet"
     />
     <link href="./public/main.css" rel="stylesheet" />
-  </head>
-  <!-- Global site tag (gtag.js) - Google Analytics -->
-  <script
-    async
-    src="https://www.googletagmanager.com/gtag/js?id=G-5KEWSX2CE2"
-  ></script>
-  <script>
-    window.dataLayer = window.dataLayer || [];
-    function gtag() {
-      window.dataLayer.push(arguments);
-    }
-    gtag("js", new Date());
+    <script type="text/javascript">
```

Comment:
Would be cleaner to put it in an external .js file and then use `window.amplitude...` below. Did you try that?

Reply:
Trying now...